.storybook/static/preview-head.js

```diff
@@ -375,3 +375,44 @@ function toggleNestedListSubmenu(event) {
         icon.classList = 'sap-icon--navigation-down-arrow';
     }
 }
+
+function collectionHas(a, b) { //helper function (see below)
+    for(var i = 0, len = a.length; i < len; i ++) {
+        if(a[i] == b) return true;
+    }
+    return false;
+}
+
+function toggleCondensedVerticalNavSubmenu(event) {
+    let button = event.target;
+
+    while (button.id !== 'parentCalendarButton') {
+        button = button.parentNode;
+    }
+
+    if(!button.classList.contains('is-expanded')) {
+        button.classList.add('is-expanded');
+    } else if(button.classList.contains('is-expanded')) {
+        button.classList.remove('is-expanded');
+    }
+}
+
+function toggleVerticalNavSubmenu(event) {
+    let button = event.target;
+    if (button.tagName.toLowerCase() !== 'li') {
+        button = event.target.parentNode;
+    }
+    let arrowIcon = button.querySelector('.fd-list__navigation-item-arrow');
+
+    if(arrowIcon && arrowIcon.classList.contains('is-expanded')) {
+        button.classList.remove('is-expanded');
```

Comment:
I think this line could be extracted into separate function

Reply:
I don't think it's necessary for 1 line of code

Reply:
Just the `button.classList.remove('is-expanded');` part? Seems a bit overkill to me